build.gradle.kts

```diff
@@ -46,6 +54,7 @@ dependencies {
     // insert your external dependencies
     // TODO actually use dependency in library
     implementation("org.apache.commons:commons-math3:3.6.1")
+    implementation("org.json:json:20240303")
```

Comment:
This is just for the example library correct?

Reply:
yes, the remote dependencies are for the example only. But even so, didn't mean to leave the json in there. thanks.